active_directory_201.rst

```diff
@@ -20,10 +20,81 @@ DHCP
 Global Catalog
 --------------
 
+The Global Catalog (GC) holds a subset of attributes for all objects in a multi-domain forest.
+It can be used to search all objects in the forest.
+For example, when searching for the a specific user, the GC can be searched to find users not only in the current domain but also in other domains in the forest.
+The attributes included in the GC's subset of object attributes are those required by the schema and those most commonly used in searches.
+Attributes can be added and removed from the Global Catalog using the Active Directory Schema snap-in.
 
-Advanced Active Directory Maintenance
-=====================================
+When Active Directory Domain Services is installed,  the GC is added to the first domain controller in the forest.
+It can be added to additional domain controllers.
```

Comment:
I would try to rewrite this and the following line, since the above is a statement of an option without "why", and the following explains the why.
Maybe remove the above, and after the below line, another one pointing the reader to the basics of "how".